runner/src/mill/runner/MillMain.scala

```diff
@@ -162,7 +162,8 @@ object MillMain {
             (false, RunnerState.empty)
 
           case Right(config) =>
-            val colored = config.color.getOrElse(mainInteractive)
+            val noColorViaEnv = env.get("NO_COLOR").exists(v => v != "" && v != "0")
```

Comment:
Spec (https://no-color.org/) says, the value must be non empty.

> Command-line software which adds ANSI color to its output by default should check for a NO_COLOR environment variable that, when present and not an empty string (regardless of its value), prevents the addition of ANSI color.

Technically even a `"0"` should enable the `NO_COLOR` request.

Reply:
Yes...I found it a bit odd, as other env variables often have a way to disable it as well with a explicit value. Zero made sense to me.

Changed to follow the spec ;).